cfme/tests/test_rest.py

```diff
@@ -26,7 +26,7 @@
     scope="module"
 )
 
-pytestmark = [pytest.mark.ignore_stream("5.2")]
+pytestmark = [pytest.mark.ignore_stream("5.3")]
```

Comment:
Ok so we can definitely ignore `5.3` which means that we should delete all the other `5.3` related stuff as well.